pkg/config/datafileprojectconfig/mappers/feature_test.go

```diff
@@ -43,13 +43,16 @@ func TestMapFeatures(t *testing.T) {
 	rolloutMap := map[string]entities.Rollout{
 		"41111": rollout,
 	}
-	experiment31111 := entities.Experiment{ID: "31111"}
-	experiment31112 := entities.Experiment{ID: "31112"}
+	experiment31111 := entities.Experiment{ID: "31111", IsFeatureExperiment: false}
```

Comment:
nit:  the zero-value for `IsFeatureExperiment` should already be false so there is no need to set it explicitly.